x-pack/plugins/security_solution/common/api/timeline/model/api.ts

```diff
@@ -8,24 +8,61 @@
 import * as runtimeTypes from 'io-ts';
 import { PositiveInteger } from '@kbn/securitysolution-io-ts-types';
 
-import { stringEnum, unionWithNullType } from '../../utility_types';
+import { stringEnum, unionWithNullType } from '../../../utility_types';
```

Comment:
We should break this file out into individual model files in this path and export it in the index similar to how you've set up the sub-domains. Will create a ticket for it!